internal/protoc/compiler.go

```diff
@@ -168,31 +164,27 @@ func (c *compiler) Compile(protoSets ...*file.ProtoSet) (*CompileResult, error)
 	}, nil
 }
 
-func (c *compiler) ProtocCommands(protoSets ...*file.ProtoSet) ([]string, error) {
-	var cmdMetaStrings []string
-	for _, protoSet := range protoSets {
-		// we end up calling the logic that creates temporary files for file descriptor sets
-		// anyways, so we need to clean them up with cleanCmdMetas
-		// this logic could be simplified to have a "dry run" option, but ProtocCommands
-		// is more for debugging anyways
-		cmdMetas, err := c.getCmdMetas(protoSet)
-		if err != nil {
-			return nil, err
-		}
-		for _, cmdMeta := range cmdMetas {
-			cmdMetaStrings = append(cmdMetaStrings, cmdMeta.String())
-		}
-		cleanCmdMetas(cmdMetas)
+func (c *compiler) ProtocCommands(protoSet *file.ProtoSet) ([]string, error) {
+	// we end up calling the logic that creates temporary files for file descriptor sets
+	// anyways, so we need to clean them up with cleanCmdMetas
+	// this logic could be simplified to have a "dry run" option, but ProtocCommands
```

Comment:
nit: Now that this uses the `dry-run` flag, could we remove this comment? My fault for not handling in https://github.com/uber/prototool/pull/135

Reply:
Can't remove this yet, needs some refactoring, this still creates temporary files.